src/app/api/contributors/team/fetchTeamContributors.ts

```diff
@@ -27,3 +27,38 @@ export async function fetchTeamContributors(
   });
   return teamContributorDto;
 }
+
+export async function fetchTeamScoreSum(teamId: string): Promise<number> {
+  const foundTeamContributors = await prisma.userScore.findMany({
+    where: {
+      user_type: "USER",
+      contribution_calculation: {
+        Team: {
+          id: teamId,
+        },
+      },
+    },
+  });
+
+  let score = 0;
+  foundTeamContributors.forEach((foundTeamContributor) => {
+    score += Number(foundTeamContributor.score);
+  });
+  return score;
+}
+
+export async function fetchTeamContributorsSum(
+  teamId: string,
+): Promise<number> {
+  const foundTeamContributors = await prisma.userScore.findMany({
+    where: {
+      user_type: "USER",
+      contribution_calculation: {
+        Team: {
+          id: teamId,
+        },
+      },
+    },
+  });
+  return foundTeamContributors.length;
+}
```

Comment:
what do you think about merging this function with the above?

They do the same query and get the same data, with the difference that the above one does the forEach map.

This way we avoid one database query

Reply:
Think its a good idea to keep them as separate functions.  We can optimise the at a later stage